api/specs/webserver/scripts/openapi_nih_sparc.py

```diff
@@ -89,14 +89,14 @@ async def get_redirection_to_viewer(
 
 
 @app.get(
-    "/study/{study_id}",
+    "/study/{id}",
     tags=TAGS,
     response_class=RedirectResponse,
     response_description="Opens osparc and opens a copy of publised study",
     status_code=status.HTTP_302_FOUND,
     operation_id="get_redirection_to_study_page",
 )
-async def get_redirection_to_study_page(study_id: ProjectID):
+async def get_redirection_to_study_page(id: ProjectID):
```

Comment:
very minor: this is a little bit weird. I guess you preferred id over study_id...

Reply:
yes, because in the near future that ide will be a permalink auto-generated that have nothing to do with the study-id